packages/LUIS/test/luis.cli.test.suite.js

```diff
@@ -41,4 +42,6 @@ describe('The LUIS cli tool', () => {
         });
         
     });
+
+    runTests(__dirname);
```

Comment:
I'd suggest running tests within a single file. Meaning, not all json files but a relative single .json filename passed as parameter.